src/appsco-app.html

```diff
@@ -667,6 +663,11 @@
 
                             <appsco-reports-page-actions id="appscoReportsPageActions"
                                                         name="reports"></appsco-reports-page-actions>
+
+                            <appsco-access-report-page id="appscoAccessReportPage" name="reports" on-open-access-report="_">
```

Comment:
Page can't be rendered here. These are page-actions elements, not pages. Also, you can't have two pages with the same name. And there is no event listener for appsco-access-report-page.